src/components/Button/Button.module.scss

```
@@ -0,0 +1,171 @@
+.sign {
+  background: $color-violet-5534da;
+  width: 32.5rem;
+  height: 3rem;
+  border-radius: 0.5rem;
+  font-size: 1.25rem;
+  font-weight: 500;
+  color: $color-white-ffffff;
+
+  @include mobile {
+    width: 22rem;
+  }
+}
+
+.sign:disabled {
+  background: $color-gray-9fa6b2;
+}
```

p4; 이렇게도 가능하지 않을까 싶어요!

```suggestion
.sign {
  background: $color-violet-5534da;
  width: 32.5rem;
  height: 3rem;
  border-radius: 0.5rem;
  font-size: 1.25rem;
  font-weight: 500;
  color: $color-white-ffffff;
  
  &:disabled {
  background: $color-gray-9fa6b2;
}

  @include mobile {
    width: 22rem;
  }
}
```